Frontend/ui-driver/src/Screens/DriverProfileScreen/ComponentConfig.purs

```diff
@@ -264,3 +269,130 @@ primaryButtonConfig1 state = let
       , alpha = 1.0
       }
   in primaryButtonConfig'
+
+activateAndDeactivateRcPopUpCongif :: forall w. (Action -> Effect Unit) ->  ST.DriverProfileScreenState -> PopUpModal.Config
+activateAndDeactivateRcPopUpCongif push state =
+  let
+    config' = PopUpModal.config
+    popUpConfig' =
+      config'
+        { primaryText { text = if state.data.rcState then (getString DEACTIVATE_RC) else (getString ACTIVATE_RC)}
```

Comment:
Can we change the name of rcState to isActiveRC, since rcState as boolean can't tell the state